spec/serializable/serializer_spec.rb

```diff
@@ -198,6 +198,22 @@ def custom_attributes
           ]
         }
       end
+
+      it "includes a whitelist of attributes if specified" do
+        serializer.as_json(person, { attribute_whitelist: [:name, :description] }).should == {
+          name: 'Gavin',
+          description: 'This is person #123',
+          custom_key: 'custom value for model id 123'
+        }
+      end
+
+      it "excludes a blacklist of attributes if specified" do
+        serializer.as_json(person, { attribute_blacklist: [:name, :description] }).should == {
+          id: '123',
+          href: '/people/123',
+          custom_key: 'custom value for model id 123'
```

Comment:
`custom_key` is always included as it's part of [`custom_attributes`](https://github.com/RestPack/restpack_serializer/pull/40) which take precedence